lib/modules/manager/npm/post-update/lerna.ts

```diff
@@ -115,18 +115,21 @@ export async function generateLockFiles(
       extraEnv.NPM_AUTH = env.NPM_AUTH;
       extraEnv.NPM_EMAIL = env.NPM_EMAIL;
     }
-    const lernaVersion =
+    const lernaConstraint =
       config.constraints?.lerna ??
-      getLernaVersion(lernaPackageFile, await lazyPgkJson.getValue());
+      getLernaConstraint(lernaPackageFile, await lazyPgkJson.getValue());
     if (
-      !is.string(lernaVersion) ||
-      (semver.valid(lernaVersion) && semver.gte(lernaVersion, '7.0.0'))
+      !is.string(lernaConstraint) ||
+      (semver.valid(lernaConstraint) && semver.gte(lernaConstraint, '7.0.0')) ||
+      (semver.validRange(lernaConstraint) &&
+        (semver.satisfies('7.0.0', lernaConstraint) ||
+          semver.satisfies('7.999.999', lernaConstraint)))
     ) {
-      logger.debug('Skipping lerna bootstrap');
+      logger.debug('Skipping lerna bootstrap for lerna >= 7.0.0');
       cmd.push(`${lernaClient} install ${cmdOptions}`);
```

Comment:
shouldn't we skip lerna global install completely to save runtime?

Reply:
This `lernaClient` is actually npm, yarn, etc. I think we do skip lerna install as-is - check the snapshots